checker/src/org/checkerframework/checker/index/samelen/SameLenAnnotatedTypeFactory.java

```diff
@@ -108,7 +114,30 @@ public AnnotationMirror createCombinedSameLen(
             bValues.addAll(SameLenUtils.getValue(sl2));
         }
 
-        return getCombinedSameLen(aValues, bValues);
+        AnnotationMirror res = getCombinedSameLen(aValues, bValues);
+        return res;
+    }
+
+    public FlowExpressions.Receiver arrayReceiverFromString(
```

Comment:
Is there anything particular to arrays in the method or is it generally useful?  I'm thinking we might want to move it to the super class. (Or perhaps into the CFAbstractTransfer class) Thoughts?